cogs/ncplayz.py

```diff
@@ -0,0 +1,51 @@
+import discord
+from discord.ext import commands
+
+class NCPlayz:
+    def __init__(self, bot):
+        self.bot = bot
+        
+    @commands.command(aliases = ["id"])
+    async def ID(self, ctx):
+        """Get the ID of a channel, user, or role"""
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Invalid Type! {0.subcommand_passed} is not a valid command!'.format(ctx))
+
+    @ID.command()
+    async def channel(self, ctx, *, channel: discord.Channel = None):
+        """Fetches the ID of a specified channel"""
+        channel = channel or ctx.channel
+        if channel:
+            msg = "The id of the channel `{0}` is `{0.id}".format(channel)
+        else: 
+            msg = "**Error!** A channel named {0} could not be found! You must enter the exact name (including caps).".format(request)
+        await ctx.send(msg)
+        
+
+    @ID.command()
+    async def role(self, ctx, *, role: discord.Role):
+        """Fetches the ID of a specified role"""
+        if role:
```

Comment:
You don't need these `if` statements. If the conversion fails, usually because it wasn't found. It raises `commands.BadArgument`.